iquip/apps/logger.py

```diff
@@ -136,41 +147,74 @@ def __init__(self, name: str, parent: Optional[QObject] = None):
         # connect signals to slots
         self.loggerFrame.clearButton.clicked.connect(self.checkToClear)
         self.confirmFrame.confirmed.connect(self.clearLog)
-        self.handler = LoggingHandler(self.addLog)
-        # TODO(aijuh): Change the log format when it is determined.
-        fs = "%(levelname)s [%(name)s] [%(filename)s:%(lineno)d] %(message)s"
-        formatter = logging.Formatter(fs)
-        self.handler.setFormatter(formatter)
+        #initialize logger
+        self.initLogger()
+        # set loggerFrame's frameLevelBox
+        self.loggerFrame.frameLevelBox.addItems(self.levels_dict.values())
+        self.loggerFrame.frameLevelBox.textActivated.connect(self.setFrameLevel)
+        self.loggerFrame.frameLevelBox.setCurrentText(self.levels_dict[self.frameHandler.level])
+        # set loggerFrame's fileLevelBox
+        self.loggerFrame.fileLevelBox.addItems(self.levels_dict.values())
+        self.loggerFrame.fileLevelBox.textActivated.connect(self.setFileLevel)
+        self.loggerFrame.fileLevelBox.setCurrentText(self.levels_dict[self.fileHandler.level])
+
+    def initLogger(self):
+        """Initializes the root logger and handlers for constructor."""
+        self.levels_dict = {10: "DEBUG", 20: "INFO", 30: "WARNING", 40: "ERROR", 50: "CRITICAL"}
```

Comment:
Please use the consistent naming convention: `levelsDict`.
I think just `levels` or `levelNames` is fine, if it is okay with you.